app/pods/components/unit-multiselect/component.ts

```diff
@@ -0,0 +1,32 @@
+import { computed, action } from '@ember-decorators/object';
+import Component from '@ember/component';
+import Unit from 'web-wars-ember/pods/unit/model';
+
+export default class UnitMultiselect extends Component {
+  public model!: Unit[];
+  public value!: string[];
+  public onChange!: (value: string[]) => void;
+  public disabled ? = false;
```

Comment:
Per my other comment
```suggestion
  public disabled = false;
```

Reply:
🙃